src/app/resources/resources.component.ts

```diff
@@ -4,12 +4,22 @@ import { DialogsDeleteComponent } from '../shared/dialogs/dialogs-delete.compone
 import { MatTableDataSource, MatPaginator, MatFormField, MatFormFieldControl, MatDialog, MatDialogRef } from '@angular/material';
 
 @Component({
-  templateUrl: './resources.component.html'
+  templateUrl: './resources.component.html',
+  styles: [ `
+    .mat-cell, .mat-header-cell {
```

Comment:
Please **do not modify existing Material classes**.  This makes the templates difficult to debug and may cause unintended side effects.